### PR TITLE
Show status details for a DC in both the deployments table and DC pages

### PR DESCRIPTION
**Pull request by jwforres**
Fixes https://github.com/openshift/origin/issues/4895

### CODE REVIEW

**Comment by jwforres** [comment]
Also fixes a js error when a dc has no deployments on the dc page

**Comment by jwforres** [comment]
@spadgett

**Comment by jwforres** [comment]
@spadgett actually hold off on review, i need to do basically the same thing for builds

**Comment by jwforres** [comment]
@spadgett actually I can't do builds yet due to a bug thats causing it to flash every second https://github.com/openshift/origin/issues/5258 so can go ahead and review this one

File: assets/app/scripts/directives/popups.js [hunk 1]
```diff
@@ -5,13 +5,44 @@ angular.module('openshiftConsole')
...
+        dynamicContent: '@?'
+      },
       link: function($scope, element, attrs) {
         if (attrs) {
           switch(attrs.toggle) {
             case "popover":
+              // If dynamic-content attr is set at all, even if it hasn't evaluated to a value
+              if (attrs.dynamicContent || attrs.dynamicContent === "") {
+                // if we actually have a value
+                if ($scope.dynamicContent) {
```
Comment (spadgett) [inline assets/app/scripts/directives/popups.js]: I could be wrong, but I think the `$watch` below fires immediately. You might be able to get rid of this `if` block.

File: assets/app/scripts/directives/popups.js [hunk 2]
```diff
@@ -5,13 +5,44 @@ angular.module('openshiftConsole')
...
+                    .popover();                
+                });                  
+              }
               $(element).popover();
               break;
             case "tooltip":
+              // If dynamic-content attr is set at all, even if it hasn't evaluated to a value
+              if (attrs.dynamicContent || attrs.dynamicContent === "") {
+                // if we actually have a value
+                if ($scope.dynamicContent) {
```
Comment (spadgett) [inline assets/app/scripts/directives/popups.js]: Same comment as above.

File: assets/app/views/browse/deployment-config.html [hunk 3]
```diff
@@ -5,7 +5,10 @@
     <div ng-if="!loaded">Loading...</div>
     <div class="tile" ng-if="deploymentConfig">
       <div> 
-        <h1>{{deploymentConfig.metadata.name}}</h1>
+        <h1>
+          {{deploymentConfig.metadata.name}}
+          <span ng-if="deploymentConfig.status.details.message" class="pficon pficon-warning-triangle-o" style="cursor: help;" data-toggle="popover" data-placement="right" data-trigger="hover" dynamic-content="{{deploymentConfig.status.details.message}}"></span>
```
Comment (spadgett) [inline assets/app/views/browse/deployment-config.html]: Is `status.details.message` always a warning?

Comment (jwforres) [inline assets/app/views/browse/deployment-config.html]: I asked, today it is always warnings

File: assets/app/scripts/directives/popups.js (hunk 1 above)
Comment (jwforres) [inline assets/app/scripts/directives/popups.js]: You were right. Seems to be working without that block.

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
bindata was bad before re[merge]

**Pull request merged**